src/frontend/specification/EnsembleParameterTable.jl

```diff
@@ -69,52 +92,53 @@ end
 ### Additional Methods
 ############################################################################################
 
-# Sorting ----------------------------------------------------------------------------------
+# Variables Sorting ------------------------------------------------------------------------
 
-# Sorting ----------------------------------------------------------------------------------
-
-function sort!(ensemble_partable::EnsembleParameterTable)
-    for partable in values(ensemble_partable.tables)
-        sort!(partable)
+function sort_vars!(partables::EnsembleParameterTable)
+    for partable in values(partables.tables)
+        sort_vars!(partable)
     end
 
-    return ensemble_partable
+    return partables
 end
 
-function sort(partable::EnsembleParameterTable)
-    new_partable = deepcopy(partable)
-    sort!(new_partable)
-    return new_partable
-end
+sort_vars(partables::EnsembleParameterTable) = sort_vars!(deepcopy(partables))
 
 # add a row --------------------------------------------------------------------------------
 
 # do we really need this?
-import Base.push!
-
-function push!(partable::EnsembleParameterTable, d::AbstractDict, group)
+function Base.push!(partable::EnsembleParameterTable, d::AbstractDict, group)
     push!(partable.tables[group], d)
 end
 
-push!(partable::EnsembleParameterTable, d::Nothing, group) = nothing
-
-# get group --------------------------------------------------------------------------------
-
-get_group(partable::EnsembleParameterTable, group) = get_group(partable.tables, group)
+Base.getindex(partable::EnsembleParameterTable, group) = partable.tables[group]
 
 ############################################################################################
 ### Update Partable from Fitted Model
 ############################################################################################
 
-# update generic ---------------------------------------------------------------------------
 function update_partable!(
-    partable::EnsembleParameterTable,
-    model_identifier::AbstractDict,
-    vec,
-    column,
+    partables::EnsembleParameterTable,
+    column::Symbol,
+    param_values::AbstractDict{Symbol},
+    default::Any = nothing,
+)
+    for partable in values(partables.tables)
+        update_partable!(partable, column, param_values, default)
+    end
+    return partables
+end
+
+function update_partable!(
+    partables::EnsembleParameterTable,
+    column::Symbol,
+    params::AbstractVector{Symbol},
+    values::AbstractVector,
+    default::Any = nothing,
 )
-    for k in keys(partable.tables)
-        update_partable!(partable.tables[k], model_identifier, vec, column)
+    param_values = Dict(zip(params, values))
+    for (id, partable) in pairs(partables.tables)
+        update_partable!(partable, column, param_values, default)
```

Comment:
Sorry, GitHub does not let me make a commitable suggestion by deleting the following lines too...
```suggestion
    return update_partable!(partables, column, Dict(zip(params, values)), default)
```